src/components/SearchBar/SearchBar.stories.tsx

```diff
@@ -3,9 +3,10 @@ import * as React from "react";
 import SearchBar from "./SearchBar";
 import Select from "../Select/Select";
 import HelperErrorText from "../HelperErrorText/HelperErrorText";
-import { IconRotationTypes, IconNames } from "../Icons/IconTypes";
 import Button from "../Button/Button";
 import { ButtonTypes, ButtonIconPositions } from "../Button/ButtonTypes";
```

Comment:
is ButtonIconPositions used anymore?

Reply:
No, we should probably get rid of it all together